website/content/economy/fred/_index.md

```diff
@@ -0,0 +1 @@
+Menu for accessing Federal Reserve Economic Datat (FRED)
```

Comment:
You need to add the TOC by default so users can expand on that!

Reply:
?